cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -219,6 +230,128 @@ def _add_conditional_storage_widgets(
                 widgets_list.append(graph_widget)
         return widgets_list
 
+    def custom_pcluster_metric(self, metric_name, filter_pattern):
+        """Adding custom metric filter from named tuple."""
+        x = aws_cdk.aws_logs.CfnMetricFilter(
+            scope=self.stack_scope,
+            id=metric_name + " Filter",
+            filter_pattern=filter_pattern,
+            log_group_name=self.cw_log_group_name,
+            metric_transformations=[
+                aws_cdk.aws_logs.CfnMetricFilter.MetricTransformationProperty(
+                    metric_namespace="ParallelCluster/Errors",
+                    metric_name=metric_name,
+                    metric_value="1",
+                    default_value=0,
+                )
+            ],
+        )
+        x.add_depends_on(self.cw_log_group)
+        return x
+
+    def add_custom_error_metrics(self):
+        """Create custom error metric filter and outputs to cloudwatch graph."""
+        jobs_not_starting_errors = [
```

Comment:
Any reason to make the error_information different from metric name?

Reply:
its like the subcases of the common errors in the intern doc